vcls/magento2/default.vcl

```diff
@@ -0,0 +1,229 @@
+vcl 4.1;
+
+import goto;
+import std;
+import ykey;
+
+# For SSL offloading, pass the following header in your proxy server or load balancer: '/* {{ ssl_offloaded_header }} */: https'
+
+backend default none;
+
+probe p1 {
+        .url = "/pub/health_check.php";
+        .timeout = 2s;
+        .interval = 5s;
+        .window = 10;
```

Comment:
This won’t mark the backend as sick until approx 20s of downtime.

Reply:
this is from upstream, I'm leaving it in